lib/Stan_math_code_gen.ml

```diff
@@ -20,22 +20,13 @@ open Core_kernel
 open Mir
 open Fmt
 
-let pp_operator = function
-  | Plus -> "+"
-  | Minus -> "-"
-  | Times -> "*"
-  | Divide -> "/"
-  | Modulo -> "%"
-  | Or -> "||"
-  | And -> "&&"
-  | Equals -> "=="
-  | NEquals -> "!="
-  | Less -> "<"
-  | Leq -> "<="
-  | Greater -> ">"
-  | Geq -> ">="
-
-let zero = Lit (Int, "0")
+let internal_expr =
+  { texpr= Var "UHOH"
```

Comment:
What's the use of this definition?

Reply:
There is a lot of boilerplate to declaring a new expression now, so it's nice to have something to generate from as you can see where this is used.